search_events_app/services/db/db_response_processor.py

```diff
@@ -42,3 +43,8 @@ def get_organizer(item):
     if item[5]:
         return item[5]
     return None
+
+
+def get_admin_url(item):
+    url = "https://www.eventbrite.com/myevent?slow_load=1&eid=" + f'{item[0]}'
```

Comment:
Don't create a new variable, just return 'https...'
Use single quotes for string and format everything in one single string like this:
f'https://www.eventbrite.com/myevent?slow_load=1&eid={item[0]}'

Reply:
Resolved!